frontends/web/src/routes/buy/components/exchangeselectionradio.tsx

```diff
@@ -30,19 +31,27 @@ type TRadioProps = RadioProps & JSX.IntrinsicElements['input'];
 
 const Deal = ({ deal }: { deal: ExchangeDealWithBestDeal }) => {
   const { t } = useTranslation();
-  const getPaymentMethodName = (methodName: ExchangeDeals['exchangeName']) => {
+
+  const renderPaymentMethod = (methodName: ExchangeDeals['exchangeName']) => {
     switch (methodName) {
     case 'bank-transfer':
-      return t('buy.exchange.bankTransfer');
+      return (<span>
+        <Bank />
+        <p className={style.paymentMethodName}>{t('buy.exchange.bankTransfer')}</p>
+      </span>);
     case 'card':
-      return t('buy.exchange.creditCard');
+      return (<span>
+        <CreditCard />
+        <p className={style.paymentMethodName}>{t('buy.exchange.creditCard')}</p>
+      </span>);
     default:
       return methodName;
     }
   };
+
   return (
     <div className={style.paymentMethodContainer}>
-      <p className={style.paymentMethodName}>{getPaymentMethodName(deal.payment)}</p>
+      {renderPaymentMethod(deal.payment)}
```

Comment:
what do you think about changing PaymentMethod so it is a function component?
it could be the in the same module and only used internally, so not exported.

Reply:
cool